icaruscode/Timing/CMakeLists.txt

```diff
@@ -2,8 +2,21 @@ cet_enable_asserts()
 
 set(    MODULE_LIBRARIES
         lardataobj::RecoBase
+	sbnobj::Common_Trigger
         larcore::Geometry_Geometry_service
         lardata::DetectorInfoServices_DetectorClocksServiceStandard_service
```

Comment:
This should not be here:
```suggestion
```
since it's an implementation of the service interface.
If there are complains (headers not found), `lardata::DetectorClocksService` might address them.

Reply:
Replaced with `lardata::DetectorClocksService` and it worked!